_drafts/SC1-4-3-text-bg-contrast.md

```diff
@@ -0,0 +1,198 @@
+---
+rule_id: SC1-4-3-text-bg-contrast
+name: Text-background contrast
+test_mode: semi-automatic
+
+criteria:
+- 1.4.3 # Contrast (Minimum) (level AA)
+
+authors:
+- Emma Pratt Richens
+- Rob Fentress
+---
+
+## Description
+
+This test checks that the text nodes on a page contrast sufficiently with the background, gradient or image behind them.
+
+## Background
+
+- [Understanding SC 1.4.3](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140311/visual-audio-contrast-contrast.html)
+- [Contrast (Minimum) Understanding SC 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
+- [G18: Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text](https://www.w3.org/TR/WCAG20-TECHS/G18)
+- [G145: Ensuring that a contrast ratio of at least 3:1 exists between text (and images of text) and background behind the text](https://www.w3.org/TR/WCAG20-TECHS/G145)
+- [F24: specifying foreground colors without specifying background colors](https://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/F24)
+- [F83: Failure of Success Criterion 1.4.3 and 1.4.6 due to using background images that do not provide sufficient contrast with foreground text (or images of text)](https://www.w3.org/TR/WCAG20-TECHS/F83.html)
+
+## Assumptions
+- Code validates to a published grammar (eg. 4.1.1 and 4.1.2)
```

Comment:
The intention was to avoid concerning ourselves with scenarios that could be cause by invalid code. I don't know that we had any explicit cases.